workflow/controller/config_controller.go

```diff
@@ -55,7 +55,7 @@ func (wfc *WorkflowController) updateConfig(cm *apiv1.ConfigMap) error {
 		}
 	}
 	wfc.session = nil
-	wfc.offloadNodeStatusRepo = nil
+	wfc.offloadNodeStatusRepo = sqldb.ExplosiveOffloadNodeStatusRepo
 	wfc.wfArchive = sqldb.NullWorkflowArchive
 	persistence := wfc.Config.Persistence
```

Comment:
We need to do nil check here

Reply:
no - at this line of code `wc.Config` cannot be nil

Reply:
ok